purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -353,29 +353,29 @@ class Purchases internal constructor(
     }
 
     /**
-     * Gets the StoreProduct for the given list of subscription skus.
-     * @param [skus] List of skus
+     * Gets the StoreProduct for the given list of subscription products.
+     * @param [productIds] List of productIds
      * @param [callback] Response callback
      */
     // TODO deprecate, replaced with getProducts
-    fun getSubscriptionSkus(
-        skus: List<String>,
+    fun getSubscriptionProducts(
```

Comment:
we should deprecate the old ones instead of fully removing them

you can search for @Deprecated annotations for examples of how to do that!